posthog/temporal/ai/session_summary/summarize_session.py

```diff
@@ -36,6 +44,61 @@
 SESSION_SUMMARIES_STREAM_INTERVAL = 0.1  # 100ms
 
 
+@temporalio.activity.defn
+async def fetch_session_data_activity(inputs: SingleSessionSummaryInputs) -> str | None:
```

Comment:
No functional changes, moved from `shared` file to reduce number of files (as I now use a separate activity for group summaries).